python/whylogs/core/metrics/metrics.py

```diff
@@ -424,6 +423,9 @@ def zero(cls, config: Optional[MetricConfig] = None) -> "DistributionMetric":
         )
 
 
+custom_metric(DistributionMetric)
```

Comment:
Can we name it something like `register_metric(DistributionMetric)` or at least a name without 'custom' if this is required for standard metrics?

Reply:
It was actually intended to be used as `@custom_metric`. I will add a `register_metric()`